.github/actions/hugo-build-versioned-action/action.yaml

```diff
@@ -35,17 +36,34 @@ runs:
       run: |
         python -m pip install --upgrade pip
         pip install -r scripts/script-requirements.txt
-    - name: Generate Versioned Documentation
+    - uses: actions/setup-go@v4
+      with:
+        go-version: '>=1.20.1'
+    - uses: actions/setup-node@v3
+      with:
+        node-version: 18
+    - name: Install dependencies
+      working-directory: ${{ inputs.working-directory }}
+      run: npm ci
+      shell: bash
+    - name: "Setup Hugo"
+      uses: peaceiris/actions-hugo@v2
+      with:
+        extended: true
+        hugo-version: '0.110.0'
+    - name: "Build"
+      env:
+        KEEP_MASTER: ${{ inputs.keep-master }}
+        THIS_BRANCH: ${{ inputs.this-branch }}
+        BASE_URL: ${{ inputs.base-url }}
+        HUGO_ENV: production
```

Comment:
Please, hide this in variable as well. I can imagine that we might want to use this action not for production.